src/compressor/Compressor.h

```diff
@@ -16,7 +16,10 @@
 #define CEPH_COMPRESSOR_H
 
 #include "include/int_types.h"
-#include "include/Context.h"
+#include "include/memory.h"
+#include "include/buffer.h"
+#include <string>
+using namespace std;
```

Comment:
maybe better off specifying the `std` namespace in this header file instead of `using namespace std` here, to avoid the namespace pollution.